qa-tests-backend/src/test/resources/application.properties

```diff
@@ -0,0 +1 @@
+logging.level.root=DEBUG
```

Comment:
From what I see in Google search, `logging.level.root` is Spring Boot's thing. Our groovy tests are not based on Spring Boot (thanks God for this!) so I'm not sure this file will have any effect.
If this file _actually_ does switch the logging level, I'd qualify that as a dark magic.

I found this another file https://github.com/stackrox/stackrox/blob/master/qa-tests-backend/src/main/resources/logback.groovy that you could use for configuration instead. I think this will be more elegant, but unfortunately you'll have to do some extra work to suppress logging from namespaces we don't own.
Here docs on this type of configuration https://logback.qos.ch/manual/groovy.html

Reply:
For some reason `logback.groovy` is not working. I switched to `logback-test.xml` and it's working correctly. I also changed config a little bit